src/Segmenter/PosSeg/PosSegmenter.cs

```diff
@@ -32,23 +33,42 @@ public class PosSegmenter
 
         #endregion
 
-        private static IDictionary<string, string> _wordTagTab;
+        private JiebaSegmenter _segmenter;
 
-        static PosSegmenter()
+        public PosSegmenter(string wordTagFile=null)
```

Comment:
public PosSegmenter(string wordTagFile=null) : this(new JiebaSegmenter(), wordTagFile)

Reply:
Changed.